go/vt/vttablet/tabletserver/tx_pool.go

```diff
@@ -141,7 +148,7 @@ func (tp *TxPool) transactionKiller() {
 		if conn.IsInTransaction() {
 			tp.txComplete(conn, tx.TxKill)
 		}
-		conn.Releasef("exceeded timeout: %v", tp.Timeout())
+		conn.Releasef("exceeded timeout: %v", timeout)
```

Comment:
nit: can use `conn.timeout` directly here in the method.

Reply:
Oops sorry I didn't implement this suggestion earlier. I saw it, but I misunderstood it 🤦